blocks/form-handler/controllers/img-upload-component-controller.js

```diff
@@ -16,7 +17,14 @@ export default function init(component, props) {
   const dropzones = component.querySelectorAll('image-dropzone');
 
   dropzones.forEach((dz) => {
-    dz.props = props;
+    dz.handleImage = async () => {
+      const file = dz.getFile();
+
+      if (!file || !(file instanceof File)) return;
+      const resp = await uploadBinaryFile(file, JSON.parse(component.dataset.configs));
```

Comment:
i feel for consistency these files should also be uploaded when user does a save.